contrib/mesos/pkg/executor/executor.go

```diff
@@ -125,16 +117,16 @@ type KubernetesExecutor struct {
 	kubeletFinished      <-chan struct{} // signals that kubelet Run() died
 	initialRegistration  sync.Once
 	exitFunc             func(int)
-	podStatusFunc        func(KubeletInterface, *api.Pod) (*api.PodStatus, error)
-	staticPodsConfig     []byte
+	podStatusFunc        func(*api.Pod) (*api.PodStatus, error)
 	staticPodsConfigPath string
 	initialRegComplete   chan struct{}
+
```

Comment:
nit: fix formatting
- remove blank line
- move comment to right side of the variable
- remove the space after the asterisk